requester.py

```diff
@@ -0,0 +1,104 @@
+import requests
+
+URL_WEATHERAPI = 'https://api.weatherapi.com/v1/forecast.json'
+URL_OPENMETEO = "https://api.open-meteo.com/v1/forecast"
+URL_METEOSTAT = 'https://meteostat.p.rapidapi.com/point/hourly'
```

Comment:
Please be consistent with the quote usage, use either `'` or `"`.

Reply:
Done

Reply:
Cool, but I cannot see any code changes?

Reply:
Now you should see.